docs/sphinx/ome-tiff/data.rst

```diff
@@ -2,202 +2,177 @@ OME-TIFF sample data
 ====================
 
 This section provides some sample data in OME-TIFF format. They include data
-produced from an acquisition system as well as artificial sample datasets (i.e.
+produced from an acquisition system as well as artificial sample datasets i.e.
 designed for developer testing that illustrate some possible data
 organizations, which should be useful if you are interested in implementing
 support for OME-TIFF within your software.
 
 All the OME-TIFF sample data discussed below are available from our
 :ometiff_downloads:`OME-TIFF sample images resource <>`.
 
-Single file OME-TIFF
---------------------
+Biological datasets
+-------------------
 
-This section lists various examples of single file OME-TIFF. For examples of
-OME-TIFF dataset distributed across multiple TIFF files, see
-:ref:`multifile_samples`.
+.. _tubhiswt_samples:
 
-.. _artificial-datasets:
+Tubhiswt
+^^^^^^^^
 
-Artificial datasets
-^^^^^^^^^^^^^^^^^^^
+The following OME-TIFF datasets consist of tubulin histone GFP coexpressing
+*C. elegans* embryos. Many thanks to
+`Josh Bembenek <http://loci.wisc.edu/people/josh-bembenek>`_ for preparing
+and imaging this sample data.
+
+The datasets were acquired on a multiphoton workstation (2.1 GHz Athlon
+XP 3200+ with 1GB of RAM) using
+`WiscScan <http://loci.wisc.edu/software/wiscscan>`_. All image
+planes were collected at 512x512 resolution in 8-bit grayscale, with an
```

Comment:
`512×512`?  This is something to maybe change wholesale though, perhaps in another PR.